docs/source/guides/configuration.rst

```diff
@@ -56,6 +59,23 @@ Environment-specific parameters
 - ``mst_enable`` enables or disables multisignature transaction support in
```

Comment:
```suggestion
 - ``mst_enable`` enables or disables multisignature transaction network transport in
```
It can be rephrased in other way, but the idea is that the flag is about transport, not the support in general.